firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/DeveloperListenerManager.java

```diff
@@ -145,10 +145,15 @@ public void removeDisplayErrorListener(
     registeredErrorListeners.remove(displayErrorListener);
   }
 
+  public void removeDismissListener(FirebaseInAppMessagingDismissListener dismissListener) {
+    registeredDismissListeners.remove(dismissListener);
+  }
+
   public void removeAllListeners() {
     registeredClickListeners.clear();
     registeredImpressionListeners.clear();
     registeredErrorListeners.clear();
+    registeredDismissListeners.clear();
```

Comment:
Do you have any tests for this code? If so, adding an extra check for the dismiss listener sounds reasonable.

Reply:
Added new tests to check the entire add remove listeners functionality and methods.